src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -3007,6 +3026,11 @@ public void watchFiles(File outputDirectory, File testOutputDirectory, final Thr
                 registerSingleFile(containerfileUsed, executor);
             }
 
+            // Always register this hidden temp directory because generateToSrc can be turned on in dev mode
```

Comment:
I don't quite follow the meaning of this comment. Also, the code below just registers the individual file and not the directory, right?

Reply:
I meant "off"